gitlab/resource_gitlab_tag_protection.go

```diff
@@ -81,6 +81,11 @@ func resourceGitlabTagProtectionRead(d *schema.ResourceData, meta interface{}) e
 
 	pt, _, err := client.ProtectedTags.GetProtectedTag(project, tag)
 	if err != nil {
+		if is404(err) {
```

Comment:
Same comment about resource not having an importer.

Reply:
same comment as well, let me know!